### PR TITLE
Ike complete order database refactor

### LINKED ISSUE #6: Customer can complete order tying payment type to order
### Context
1. What should the feature do?

This feature should allow a registered customer to use a command line interface tool for the Bangazon Platform to complete an order (containing at least one item) and select a payment method to be used for said order. 

### Associated Product Ticket

#1 

### Requirements

- Unit tests to verify order completion in accordance with a chosen payment method and user.

- Appropriate classes for orders, payment methods & users.

- Appropriate methods for completing an order & confirming payment method. 

- A method of persisting chosen payment type + account number. 

- Database table(s) to persist pertinent order information.

### PR DESCRIPTION
**Pull request by consumerike**
# Description
Complete an order by tying payment type to order and setting payment complete to 1.

## Number of Fixes
#6 , #9, #20,  

## Related Ticket(s)
#1 ,
## Problem to Solve
Describe the problem(s) this pull request solves.
Tests a customer can complete an order e.g.-Allows a customer to complete an order (add payment type to an order and set order complete to 1)
## Proposed Changes
Describe the proposed changes.

## Expected Behavior
Describe the expected behavior.
Within the database, selected order should be 1

## Steps to Test Solution

1. git fetch
2. drop database
3. go to tests folder; `run python runtests.py`

## Testing

[x ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ x] I certify that all existing tests pass

## Documentation
[ x] There is new documentation in this pull request that must be reviewed..
[ x] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

### CODE REVIEW

**Review by stevenwally — APPROVED** [review]
you the main banana 🍌 

tests pass!

**Comment by sales-ls21** [comment]
MAIN BANANAS :banana: :banana: :banana:

**Comment by ZoeLeBlanc** [comment]
works amazing!!!! RUNS ALL THE TESTS :1st_place_medal: 👍 it's bananas 🍌

File: app/orderfinalizer.py [hunk 1]
```diff
@@ -1,58 +1,127 @@
...
+                WHERE payment_complete = 1
+                AND customerId = {} 
+                """.format(cid)
+            )
+            payment_status = cursor.fetchall()[0][4]
+            return payment_status     
+    
+
+
+            # except:
```
Comment (stevebrownlee) [inline app/orderfinalizer.py:102]: Please remove commented code before submitting a PR.

File: app/productonordermanager.py [hunk 2]
```diff
@@ -47,6 +47,7 @@ def add_product_to_order(self, product):
                 AND quantity = {} 
                 """.format(product.get_product_title(), product.get_product_description(), product.get_product_price(), product.get_product_quantity()))
             selected_product = cursor.fetchone()
+            print(selected_product, "this is selected prod w zoe")
```
Comment (stevebrownlee) [inline app/productonordermanager.py:50]: Remove debugging statements before submitting a PR

File: tests/test_customer_can_tie_payment_type_to_order.py [hunk 3]
```diff
@@ -3,53 +3,75 @@
...
+        with sqlite3.connect('../bangazon.db') as conn:
+            cursor = conn.cursor()
+            cursor.execute("""
+                SELECT * FROM Products
+                WHERE productId = {}
+                """.format(4))
+            selected_product = cursor.fetchone()
+            cursor.close()
+
+        rfk = Customer("Robert Kennedy", '1919 Heron Pointe Dr', "Nashville", "TN", 37214, '8888978880')
```
Comment (stevebrownlee) [inline tests/test_customer_can_tie_payment_type_to_order.py:60]: Some white space to break up the code into conceptual blocks, along with comments, will make this code easier to read

**Pull request merged**